crypto3/readme.md

```diff
@@ -1,132 +1,61 @@
-# =nil; Foundation's Cryptography Suite
-[![Build docs](https://github.com/NilFoundation/crypto3/actions/workflows/build_docs.yaml/badge.svg)](https://github.com/NilFoundation/crypto3/actions/workflows/build_docs.yaml) 
-[![Discord](https://img.shields.io/discord/969303013749579846.svg?logo=discord&style=flat-square)](https://discord.gg/KmTAEjbmM3)
-[![Telegram](https://img.shields.io/badge/Telegram-2CA5E0?style=flat-square&logo=telegram&logoColor=dark)](https://t.me/nilfoundation)
-[![Twitter](https://img.shields.io/twitter/follow/nil_foundation)](https://twitter.com/nil_foundation)
-
+# Cryptography Suite
 Crypto3 cryptography suite's purpose is:
 1. To provide a secure, fast and architecturally clean C++ generic cryptography schemes implementation.
 2. To provide a developer-friendly, modular suite, usable for novel schemes implementation and further
    extension.
 3. To provide a Standard Template Library-alike C++ interface and concept-based architecture implementation.
 
 Libraries are designed to be state of the art, highly performant and providing a one-stop solution for
-all cryptographic operations. They are supported on all operating systems (*nix, windows, macOS)
-and architectures(x86/ARM).
+all cryptographic operations. They are supported on Linux operating system and architectures (x86/ARM).
 
 Developed by [=nil; Crypto3](https://crypto3.nil.foundation) and supported by [=nil; Foundation](https://nil.foundation).
 
 Rationale, tutorials and references are available [here](https://crypto3.nil.foundation/projects/crypto3)
  
 ## Contents
-1. [Repository Structure](#repository-structure)
-2. [Installation](#installation)
-3. [Usage](#usage)
-3. [Contributing](#contributing)
-4. [Community](#community)
+1. [Structure](#structure)
+2. [Build & test](#build_&_test)
+3. [Usage](#uage)
 
-## Repository Structure
+## Structure
 This repository is an umbrella-repository for the whole suite. Single-purposed libraries repositories (e.g. [block
```

Comment:
Maybe remove this paragraph at all? Crypto3 is not a repository anymore, and there is no way to build single library now.

Reply:
It useful for quick understand content and code structure.
Keep only in the root readme not so convenient - will be huge, so added for all second level sub folders